src/App.js

```diff
@@ -12,23 +12,85 @@ import NewGameButton from './components/NewGameButton/NewGameButton'
 
 import data from './data/data.json'
 /** actions  */
-import { setWordToGuess } from './state/actions'
+import { setWordToGuess, resetGame } from './state/actions'
 import { selectWordToGuess } from './helpers/utils'
 
 
 const App = () => {
   const dispatch = useDispatch()
-  const { isGameStarted } = useSelector(state => state.cthuluReducer)
+  const { isGameStarted, displayValues, wordToGuess, guessCount, guessedLetters } = useSelector(state => state.cthuluReducer)
+
+  const [chances, setChances] = useState(0)
+  const [removeChance, setRemoveChance] = useState(false)
+
+  useEffect(() => {
+    startGame()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
 
   useEffect(() => {
+    console.log('< COUNT > ', guessCount, guessedLetters)
+    /** check arrays */
+    if (isGameStarted) compareArrays(displayValues, wordToGuess)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [displayValues, wordToGuess, guessCount])
+
+  const compareArrays = (arrayToComplete, arrayAnswer) => {
+    const array1 = arrayToComplete.toString()
+    const array2 = arrayAnswer.toString()
+
+    const lastLetterAdded = guessedLetters[0]
+    /** if last letter is wrong, remove a chance */
+    if ( !arrayAnswer.includes(lastLetterAdded) ) {
+      /** decrease chances */
+      console.log('< remove a chance > ' )
+      setRemoveChance(true)
+      if (chances > 0) setChances(chances - 1)
+      setTimeout(() => {
+        setRemoveChance(false)
+        /** call end game method */
+        if (isGameStarted && chances === 1) gameOverAction()
+      }, 1000)
+    }
+
+
+    console.log('< VALUES > ', array1, array2, array1===array2, arrayToComplete, arrayAnswer)
+    return array1 === array2 ? gameWinAction() : false
+  }
+
+
+  const startGame = () => {
     const wordToGuess = selectWordToGuess(data, window.innerWidth)
     dispatch( setWordToGuess(wordToGuess) )
-  }, [dispatch])
+    /** create chances */
+    setChances((Number(wordToGuess.length) * 1.5 < 26 ) ? Math.floor(wordToGuess.length * 1.5)  : 23)
+  }
+
+  const gameOverAction = () => {
+    alert('YOU LOSE, try again!')
+    dispatch( resetGame() )
+    startGame()
+  }
+
+  const gameWinAction = () => {
+    alert('< YOU WOON, GET MY PERK!! >')
+
+    const wordToSearch = displayValues.reduce((acc, cur) => acc + cur, '')
+
+    window.open(`https://www.google.com/search?q=${wordToSearch} cthulhu`)
```

Comment:
Brilliant, but let's hone this down to wikipedia or monster wiki or something we know will have all the definitions available

Reply:
Well, could you help me with some example links? I really dont know if wikipedia covers all the monsters in game.

Reply:
Did a bit of testing on this one and without some refactoring it'll be hard to direct link these outside of google. Sorry bout that, you can change the linking back to google or leave it as is and I'll fix it